packages/ui/src/components/commit-copy-actions.tsx

```diff
@@ -1,12 +1,10 @@
 import { useEffect, useState } from 'react'
 import { Link } from 'react-router-dom'
 
+import { Icon, ShaBadge, Text } from '@/components'
+import { cn } from '@/utils/cn'
```

Comment:
Super minor, but you can drop the first `/`
```suggestion
import { cn } from '@utils/cn'
```